addons/token-exchange/rook_secret_handler.go

```diff
@@ -26,7 +26,7 @@ type rookSecretHandler struct {
 const (
 	RookType                                  = "kubernetes.io/rook"
 	RookDefaultBlueSecretMatchConvergedString = "cluster-peer-token"
-	RookDefaultBlueSecretMatchExternalString  = "rook-ceph-mon"
+	DefaultExternalSecretName                 = "rook-ceph-mon"
```

Comment:
Renaming is not a part of the fix. Do we need it?

Reply:
Not necessary but good to have :). the original name is convoluted